src/python/AsyncStageOut/PublisherWorker.py

```diff
@@ -639,26 +639,23 @@ def format_file_3(self, file):
         return nf
 
 
-    def migrateByBlockDBS3(self, workflow, migrateApi, destReadApi, sourceApi, inputDataset, inputBlocks = None):
+    def migrateByBlockDBS3(self, workflow, migrateApi, destReadApi, sourceApi, dataset, blocks = None):
         """
         Submit one migration request for each block that needs to be migrated.
-        If inputBlocks is missing, migrate the full dataset.
-        :return: - the output of destReadApi.listDatasets(dataset = inputDataset)
-                   if all migrations have finished ok,
-                 - an empty list otherwise.
+        If blocks is missing, migrate the full dataset.
```

Comment:
It means "If the caller didn't specify the input parameter 'blocks' (which would be a sublist of blocks from the dataset), then assume we have to migrate the whole dataset." I can change that comment line to "If the argument 'blocks' is not specified, migrate the whole dataset." or something alike (feel free to recommend).